core/src/main/java/com/klaytn/caver/contract/ContractWrapper.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2021 The caver-java Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the “License”);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an “AS IS” BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.klaytn.caver.contract;
+
+import com.klaytn.caver.Caver;
+
+import java.io.IOException;
+
+/**
+ * Representing a ContractWrapper
+ * 1. This class wraps constructors of Contract class
+ * 2. This class should be accessed via `caver.contract`
+ */
+public class ContractWrapper {
+    /**
+     * A Caver instance
+     */
+    private Caver caver;
+
+    public ContractWrapper(Caver caver) {
```

Comment:
Please add a description.

Reply:
@sirano11 
Oops... I missed it T_T
Thanks for your feedback. I'll add description.

Reply:
I added a description with this [commit](https://github.com/klaytn/caver-java/pull/221/commits/42a2fbe5b1d57a6b38b88611b0336795818ed913).
Thanks, @sirano11 again.